### PR TITLE
add notification after mail was sended

### PR DESCRIPTION
**Pull request by mfka**

### CODE REVIEW

File: src/components/Sender/Sender.tsx [hunk 1]
```diff
@@ -40,7 +42,12 @@ const Sender = () => {
 
   const sendEmails = () => {
     if (user !== undefined) {
-      send(dataToSend, user);
+      for (const recipient of dataToSend) {
+        send(recipient, user);
+        props.enqueueSnackbar(`Email to: ${recipient.email} was sended!`, {
```
Comment (plorencrstit) [inline src/components/Sender/Sender.tsx]: `was sent`

**Comment by mfka** [comment]
@bartekczyz dzisiaj spróbuję dopisać testy :)

File: src/components/Sender/Sender.tsx [hunk 2]
```diff
@@ -1,5 +1,7 @@
 import { Button, Grid, TextField } from '@material-ui/core';
-import React, { memo, useContext, useState } from 'react';
+import { withSnackbar } from 'notistack';
+import PropTypes from 'prop-types';
```
Comment (bartekczyz) [inline src/components/Sender/Sender.tsx]: 🐛 Don't use 'prop-types' with TypeScript

File: src/components/Sender/Sender.tsx [hunk 3]
```diff
@@ -18,7 +20,7 @@ const styles = {
   },
 };
 
-const Sender = () => {
+const Sender = (props: any) => {
```
Comment (bartekczyz) [inline src/components/Sender/Sender.tsx]: 🐛 Props are not `any`. Define props as follows:

```ts
import { withSnackbar, withSnackbarProps } from 'notistack';

// ...

type Props = withSnackbarProps;

const Sender = (props: Props) => {
// ...
```

File: src/components/Sender/Sender.tsx [hunk 4]
```diff
@@ -60,16 +67,15 @@ const Sender = () => {
...
     </Grid>
   );
 };
 
-export default memo(Sender);
+Sender.propTypes = {
+  enqueueSnackbar: PropTypes.func.isRequired,
+};
+
+export default withSnackbar(Sender);
```
Comment (bartekczyz) [inline src/components/Sender/Sender.tsx]: 🐛 This should be memoized

```suggestion
export default memo(withSnackbar(Sender));
```

File: src/App.tsx [hunk 5]
```diff
@@ -26,15 +27,17 @@ function App() {
...
-        </UserCtx.Provider>
-      </MuiThemeProvider>
-    </ThemeProvider>
+    <SnackbarProvider>
+      <ThemeProvider theme={theme}>
+        <MuiThemeProvider theme={theme}>
+          <UserCtx.Provider value={{ user, setUser, removeUser }}>
+            <CssBaseline />
+            <Bar />
+            {user ? <Steps /> : null}
```
Comment (bartekczyz) [inline src/App.tsx]: 🤷‍♂️ By the way you can simplify this

```suggestion
            {user && <Steps />}
```

File: src/components/Sender/Sender.tsx [hunk 6]
```diff
@@ -60,16 +67,15 @@ const Sender = () => {
...
-      >
+      <Button variant="contained" color="primary" onClick={sendEmails}>
         Send
       </Button>
     </Grid>
   );
 };
 
-export default memo(Sender);
+Sender.propTypes = {
```
Comment (bartekczyz) [inline src/components/Sender/Sender.tsx]: 🐛 Remove all `propTypes` references

**Review by bartekczyz — CHANGES_REQUESTED** [review]
Couple of changes are required

**Review by bartekczyz — APPROVED** [review]
Let's merge it @mfka

**Pull request closed without merging**